pkg/controller/util.go

```diff
@@ -757,8 +757,12 @@ func GetPreallocation(client client.Client, dataVolume *cdiv1.DataVolume) bool {
 // GetClusterWideProxy returns the OpenShift cluster wide proxy object
 func GetClusterWideProxy(r client.Client) (*ocpconfigv1.Proxy, error) {
 	clusterWideProxy := &ocpconfigv1.Proxy{}
-	if err := r.Get(context.TODO(), types.NamespacedName{Name: ClusterWideProxyName}, clusterWideProxy); err != nil {
-		return nil, err
+	// Ignore both no CRD found (IgnoreIsNoMatch) and the object itself not existing IsNotFound because we want to skip if not
+	// in Open Shift.
+	if err := r.Get(context.TODO(), types.NamespacedName{Name: ClusterWideProxyName}, clusterWideProxy); IgnoreIsNoMatchError(err) != nil {
```

Comment:
I am not sure If I like this way, that one error is checked inside the if (line 763) and another on line 762
In my opinion it would be easier to read if both are on the same level

Reply:
Its 6 of one half a dozen of the other, but I can certainly put it on the same line.